package.json

```diff
@@ -46,7 +46,8 @@
     "prepublish": "npm run build",
     "review": "eslint src/**",
     "fix": "eslint --fix src/**",
-    "test": "mocha"
+    "test": "mocha",
+    "coverage": "node_modules/.bin/babel-node node_modules/.bin/babel-istanbul cover node_modules/.bin/_mocha -- test/*.js"
```

Comment:
@zhenyab when calling in `npm.scripts`, you do not need the `nod_modules/.bin` prefix on the paths, so this should work (but not checked):

`babel-node babel-istanbul cover _mocha -- test/*.js`